lib/model/vertex.js

```diff
@@ -62,8 +104,24 @@ var Vertex = createClass({
       initOnly: true
     },
 
+    /**
+     * Key/value pairs of styles
+     * Guaranteed to have default styles set
+     * (which come from the static DEFAULT_STYLES on Vertex)
+     * @attribute styles
+     * @type      Object
+     */
     styles: {
-      //dunno
+      validator: Lang.isObject,
+      valueFn: function() {
+        return Vertex.DEFAULT_STYLES;
+      },
+      
+      setter: function(val) {
+        // merge set styles w/ defaults
+        // so that it's guaranteed to have defaults set
+        return ObjectUtils.merge(Vertex.DEFAULT_STYLES, val);
+      }
```

Comment:
True. Maybe we have objects for each block of GSL, and every style-able object maintains a list of these blocks with a matching selector. It can then be computed on the fly, maybe with caching. 
Either way, I think I'm going to change this basic object to a class, which will act as a facade to whatever way we end up doing it.